### PR TITLE
Update EditAction.js

### PR DESCRIPTION
**Pull request by M1racle04**
Hi this issue was reserved for me and I have done the necessary changes

Fixes #0000 (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `grunt test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updates
* [ ] @mention the original creator of the issue in a comment below for help or for a review

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

File: src/edit/actions/EditAction.js [hunk 1]
```diff
@@ -79,12 +79,12 @@ L.EditAction = L.Toolbar2.Action.extend({
...
   },
 
-  _enableAction: function() {
+  _enableAction() {
     L.DomUtil.removeClass(this._link.parentElement, 'disabled');
     L.DomUtil.removeClass(this._link, 'disabled');
   },
 
-  _disableAction: function() {
+ _disableAction() {
```
Comment (TildaDares) [inline src/edit/actions/EditAction.js:87]:
```suggestion
  _disableAction() {
```

File: src/edit/actions/EditAction.js [hunk 2]
```diff
@@ -66,7 +66,7 @@ L.EditAction = L.Toolbar2.Action.extend({
     this._addSubToolbar(toolbar, this._icon, args);
   },
 
-  _injectIconSet: function() {
+ _injectIconSet() {
```
Comment (TildaDares) [inline src/edit/actions/EditAction.js:69]:
```suggestion
  _injectIconSet() {
```

**Review by TildaDares — CHANGES_REQUESTED** [review]
Hi @M1racle04, feel free to just click the `Commit Suggestion` button.

**Review by TildaDares — APPROVED** [review]
Great work @M1racle04. Thank you!!

**Comment by M1racle04** [comment]
Thank You so much

**Pull request merged**